### ISSUE TITLE
Lost LUNC using Terra Bridge with shuttle to Terra Station 

### ISSUE THREAD

**Issue by sassanomi** [terra-money/bridge-web-app#219]
Hello

my LUNC did not arrive to Terra Station
I've used Terra Bridge with shuttle, by connecting my meta mask wallet...ut nothing has arrived.

Transaction Hash:
0x7ce0c5fc0e1ddf63ea9272cf8555cf118fb88be778770985fd59b5b281784131

Wallet from: 0xB866a8bd02F8DDEe0a499aFfa663b70cb0D1e5B3
Wallet to: terra1hh85yp2e4gven0dq4k4p4hfgphw8vfwrtdzjhm

Could you please help me?

Thanks a lot

**Comment by votrix81** [comment]
Did you figure this out? I have the same problem!

**Comment by sassanomi** [comment]
Yes, I had just to wait 5-6 days... and then all LUNC were in Terra Station
